backend/de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/model/M_Element.java

```diff
@@ -165,16 +165,15 @@ protected boolean afterSave(final boolean newRecord, final boolean success)
 
 	private void updateDependentADEntries()
 	{
-		final IElementTranslationBL elementTranslationBL = Services.get(IElementTranslationBL.class);
-
 		final AdElementId adElementId = AdElementId.ofRepoId(getAD_Element_ID());
 		if (is_ValueChanged(COLUMNNAME_ColumnName))
 		{
 			final String columnName = getColumnName();
-		elementTranslationBL.updateColumnNameFromElement(adElementId, columnName);
+			Services.get(IADTableDAO.class).updateColumnNameByAdElementId(adElementId, columnName);
+			Services.get(IADProcessDAO.class).updateColumnNameByAdElementId(adElementId, columnName);
 		}
 
-
+		final IElementTranslationBL elementTranslationBL = Services.get(IElementTranslationBL.class);
```

Comment:
is elementTranslationBL still needed?

Reply:
It is, because some rows below there's the call for element trls propagation : 

```java
elementTranslationBL.propagateElementTrls(adElementId, baseADLanguage);
```